lib/elements/stroke.py

```diff
@@ -472,7 +472,12 @@ def to_stitch_groups(self, last_patch):
                 path = [Point(x, y) for x, y in path]
                 # manual stitch
                 if self.manual_stitch_mode:
-                    patch = StitchGroup(color=self.color, stitches=path, lock_stitches=(None, None))
+                    patch = StitchGroup(color=self.color,
+                                        stitches=path,
+                                        lock_stitches=self.lock_stitches,
+                                        trim_after=self.trim_after,
+                                        stop_after=self.stop_after,
+                                        force_lock_stitches=self.force_lock_stitches)
```

Comment:
I'd really like to avoid just enabling lock stitches here, since I can't think of a good way to handle existing SVGs.  How about this?

```suggestion
                    if self.force_lock_stitches:
                        lock_stitches = self.lock_stitches
                    else:
                        # manual stitch disables lock stitches unless they force them
                        lock_stitches = (None, None)
                    patch = StitchGroup(color=self.color,
                                        stitches=path,
                                        lock_stitches=lock_stitches,
                                        force_lock_stitches=self.force_lock_stitches)
```

There's a subtlety here: we're reusing the `force_lock_stitches` setting, which normally will enforce lock stitch at the end of this stitch group even if the next stitch group in the stitch plan is not far enough away to warrant them.  That logic will still apply since we're passing `self.force_lock_stitches` through.  I think that's fine, since in this case they're specifically seeking to make lock stitches happen when they normally wouldn't on manual stitch.

Thoughts?